src/components/common/Extensions/MainNavigation/partials/hooks.ts

```diff
@@ -0,0 +1,68 @@
+import { useMemo } from 'react';
+import { NavItem } from './types';
```

Comment:
Add a blank space between the imports - I think import packages and other imports should be separated by the blank line for better readability.

Reply:
Fixed